### ISSUE TITLE
Zxing not processing barcode using iPad Camera

### ISSUE THREAD

**Issue by lalithswarna** [zxing/zxing#982]
I have been evaluating libraries that will help me scan a barcode using iPad Camera. Having issues to scan properly. My sample application invokes iPad camera and sends the image data to Zxing Api for processing
**Issues**
1.) Zxing not processing images from camera, Attached images from iPad camera that are not getting processed. Response is null.
3.) Unable to process barcode if the image contains other content for eg: any document that has a barcode inside with other text or images
2.) Slightly disoriented images not getting processed.
*Attached few images below 
**Zxingjs**
Created a sample application that has below :
ZxingClient.html : This invokes the iPad camera, captures the barcode image and calls zxingApi to get barcode content as response.
<!DOCTYPE html>
<html lang="en">
<script src="https://cdn.rawgit.com/serratus/quaggaJS/0420d5e0/dist/quagga.min.js"></script>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
<head>
    <title></title>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
</head>

<body>
    
    <div id="scanner-container"></div>
    <input id="myFileInput" type="file" onchange='sendPic();' accept="image/*;capture=camera">
    <br>
    <label id="lblBarcodeType"></label>
    <br>
    <label id="lblBarcodeContent"></label>

    <script>
    
            function sendPic() {
                var myInput = document.getElementById('myFileInput');
                var data = myInput.files[0];
               
               if (myInput.files.length > 0) {
                    if (window.FormData !== undefined) {
                        var data = new FormData();
                        for (var x = 0; x < myInput.files.length; x++){
                            data.append("file" + x, myInput.files[x]);
                        }

                        $.ajax({
                            type: "POST",
                            url: '/zxingapi/api/Zxing',
                            processData: false,
                            contentType: false,
                           data: data,
                            dataType: 'json',
                            success: function(result) {
                                console.log(result);
                                document.getElementById('lblBarcodeType').innerText = 'BarcodeType : ' + result.BarcodeType;
                                document.getElementById('lblBarcodeContent').innerText = 'BarcodeContent : ' + result.BarcodeContent;
                            },
                            error: function (xhr, status, p3, p4){
                                var err = "Error " + " " + status + " " + p3 + " " + p4;
                                if (xhr.responseText && xhr.responseText[0] == "{")
                                    err = JSON.parse(xhr.responseText).Message;
                                    console.log(err);
                                    }
                                });
                        } else {
                            alert("This browser doesn't support HTML5 file uploads!");
                        }
                    }
            }
    </script>
</body>
</html>
ZxingApi Sample: This application processes the image and sends reponse with barcode data.
ScanBarcode Method Code below : Returns the barcode content
  
      [HttpPost]
        public async Task<HttpResponseMessage> ScanBarcode()
        {
            HttpResponseMessage response = null;
            try
            {
                var provider = new MultipartMemoryStreamProvider();
                await Request.Content.ReadAsMultipartAsync(provider);

                // extract file name and file contents
                var fileNameParam = provider.Contents[0].Headers.ContentDisposition.Parameters
                    .FirstOrDefault(p => p.Name.ToLower() == "filename");
                string fileName = (fileNameParam == null) ? "" : fileNameParam.Value.Trim('"');
                byte[] file = await provider.Contents[0].ReadAsByteArrayAsync();
                Stream stream = new MemoryStream(file);

                var barcodeResponse = new BarcodeResponse { };
                IBarcodeReader reader = new BarcodeReader();
                var barcodeBitmap = (Bitmap)Image.FromStream(stream);
                var result = reader.Decode(barcodeBitmap);
                if (result != null)
                {
                    barcodeResponse.BarcodeType = result.BarcodeFormat.ToString();
                    barcodeResponse.BarcodeContent = result.Text;
                }
                 response = Request.CreateResponse(HttpStatusCode.OK, barcodeResponse);
            }
            catch(Exception ex)
            {

            }
            return response;
        }

Images:
![img_0683_working_after_crop](https://user-images.githubusercontent.com/10346160/38194045-d6782f9c-3692-11e8-9d3c-38108f486c96.jpg)
![img_0684_working_after_crop](https://user-images.githubusercontent.com/10346160/38194046-d6a8555a-3692-11e8-865b-04973770f0c4.jpg)
![img_not_working1](https://user-images.githubusercontent.com/10346160/38194047-d6d76638-3692-11e8-9fcb-f59f04e88051.JPG)
![img_notworking](https://user-images.githubusercontent.com/10346160/38194048-d7067a04-3692-11e8-83bc-3a28024e0d0e.JPG)
![img_notworking_after_crop](https://user-images.githubusercontent.com/10346160/38194049-d7340262-3692-11e8-8f64-949635552d13.JPG)
![img_working_after_crop](https://user-images.githubusercontent.com/10346160/38194050-d7628f06-3692-11e8-8b67-aee9d88c919f.JPG)

                                                                           --End--

**Comment by srowen** [comment]
This concerns a copy of the project code that's separate now. This isn't part of the project anymore.
